proto/asgt/v2/type/trainings_response.proto

```diff
@@ -14,7 +15,20 @@ message Training {
     google.protobuf.Timestamp finish_time = 5;
 }
 
+message ConsumerTraining {
+    asgt.type.Dataset dataset = 1;
```

Comment:
we can add dataset into `Training` message with id of `6` ... grpc is backwards compatible https://www.beautifulcode.co/blog/88-backward-and-forward-compatibility-protobuf-versioning-serialization